deploy/examples/object.yaml

```diff
@@ -109,6 +109,10 @@ spec:
       disabled: false
     readinessProbe:
       disabled: false
+  # If a CephObjectStoreUser is created in a namespace other than the Rook cluster namespace,
+  # the namespace must be added to the list of allowed namespaces, or specify "*" to allow all namespaces.
+  # allowUsersInNamespaces:
```

Comment:
this should be updated in docs also

Reply:
Which doc is missing the update? There are a couple docs updated with this property